source/Use-Protocol.rst

```diff
@@ -76,7 +89,6 @@ The following request types are lockable:
 - OP_SET
 - OP_CMD*
 - OP_SEND
```

Comment:
readers may wonder why send is an OP command but not included in the WP message operations; re-raise question of what is the difference between and Get/Set/Cmd and OP_GET/OP_SET/OP_CMD